docker/app/Dockerfile.prod

```diff
@@ -0,0 +1,25 @@
+FROM lf-app-base:prod
+
+# php customizations
+RUN mv $PHP_INI_DIR/php.ini-production $PHP_INI_DIR/php.ini
+COPY docker/app/customizations.php.ini $PHP_INI_DIR/conf.d/
+
+# apache2 customizations
+RUN a2enmod headers rewrite
+COPY docker/app/000-default.conf /etc/apache2/sites-enabled
+
+# install app dependencies
```

Comment:
The composer step seems ripe for a MULTI STAGE build since the only thing we actually need is a directory full of files that composer generates but not the tool itself.